.github/workflows/ci-angular.yaml

```diff
@@ -0,0 +1,80 @@
+name: Angular CI
+
+on:
+  push:
+    branches: [ "master" ]
+env:
+  IMAGE_NAME: angular
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+jobs:
+  testing:
```

Comment:
use short semantic names for jobs ans steps.

They should be verbs answering `What I will do?`...I will >build<, I will >test<, I will >deploy< etc.

Reply:
Fixed